### PR TITLE
fix(web): Normalize User / Project Id Types

### LINKED ISSUE #49: Normalize User / Project Id Types
Currently we have some ids stored as numbers, some as strings (see here for more details: https://github.com/uninbox/UnInbox/issues/49#issuecomment-1925246196)

Those should be normalized so we don't have to do conversions anymore when comparing them.

Also the current conversions should be removed

### PR DESCRIPTION
**Pull request by skushagra9**
## What does this PR do?

Fixes #49

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required
- [x] Read [Contributing Guide](./CONTRIBUTING.md)
- [x] Self-reviewed my own code
- [x] Tested my code in a local environment
- [ ] Commented on my code in hard-to-understand areas
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the UnInbox Docs if changes were necessary

### CODE REVIEW

**Comment by skushagra9** [comment]
In the `apps/web-app/server/utils/authjsDbAdaptor` 
image
Here, we have to use `+`, because id is hardcoded as string over here:
image

I have tried to change this , but it is interlinked , so I have to make changes in many places.
Please let me know if you still want the changes, I will add them

File: apps/web-app/server/utils/user.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export async function isUserAdminOfOrg(orgContext: OrgContext) {
   if (!orgContext?.memberId) return false;
   const userOrgMembership = orgContext?.members.find((member) => {
     //@ts-ignore
-    return +member.id === +orgContext?.memberId;
+    return member.id === +orgContext?.memberId;
```
Comment (muescha) [inline apps/web-app/server/utils/user.ts:7]:
```suggestion
    return member.id === orgContext?.memberId;
```
Did you forget the second `+`, or did you intentionally leave it here?

Comment (muescha) [reply apps/web-app/server/utils/user.ts:7]: Okay, I understand that the + symbol isn't suitable for every ID.

Comment (skushagra9) [reply apps/web-app/server/utils/user.ts:7]: yes

Comment (McPizza0) [reply apps/web-app/server/utils/user.ts:7]: @muescha is correct, the `orgContext.memberId` is already typed as a number, and so we dont need the `+ `

**Review by McPizza0 — APPROVED** [review]
good to go

**Comment by skushagra9** [comment]
by the way , the regex search you told, I was only able to find 75 results, please share how did you find other

**Comment by McPizza0** [comment]
> by the way , the regex search you told, I was only able to find 75 results, please share how did you find other

i guess it missed a few other id types
i searched for `+` and manually checked them all

**Comment by skushagra9** [comment]
Got it, thanks

**Pull request merged**